testing/integration/src/reward_actor.rs

```diff
@@ -0,0 +1,100 @@
+// Copyright 2023 Zondax AG
+// SPDX-License-Identifier: Apache-2.0, MIT
+use std::str::FromStr;
+
+use fvm_ipld_encoding::tuple::*;
+use fvm_shared::bigint::bigint_ser;
+use fvm_shared::clock::{ChainEpoch, EPOCH_UNDEFINED};
+use fvm_shared::econ::TokenAmount;
+use fvm_shared::math::PRECISION;
+use fvm_shared::sector::{Spacetime, StoragePower};
+use fvm_shared::smooth::FilterEstimate;
+use lazy_static::lazy_static;
+
+lazy_static! {
+    pub static ref BASELINE_EXPONENT: StoragePower =
+        StoragePower::from_str("340282591298641078465964189926313473653").unwrap();
+    /// 36.266260308195979333 FIL
+    pub static ref INITIAL_REWARD_POSITION_ESTIMATE: TokenAmount = TokenAmount::from_atto(36266260308195979333u128);
+    /// -1.0982489*10^-7 FIL per epoch.  Change of simple minted tokens between epochs 0 and 1.
+    pub static ref INITIAL_REWARD_VELOCITY_ESTIMATE: TokenAmount = TokenAmount::from_atto(-109897758509i64);
+
+    pub static ref BASELINE_INITIAL_VALUE: StoragePower = StoragePower::from(2_888_888_880_000_000_000u128);
+
+    /// 1EiB
+        pub static ref INIT_BASELINE_POWER: StoragePower =
+        ((BASELINE_INITIAL_VALUE.clone() << (2*PRECISION)) / &*BASELINE_EXPONENT) >> PRECISION;
+
+    /// 330M for mainnet
+    pub(super) static ref SIMPLE_TOTAL: TokenAmount = TokenAmount::from_whole(330_000_000);
+    /// 770M for mainnet
+    pub(super) static ref BASELINE_TOTAL: TokenAmount = TokenAmount::from_whole(770_000_000);
+}
+
+/// Reward actor state
+#[derive(Serialize_tuple, Deserialize_tuple, Default, Debug, Clone)]
+pub struct State {
```

Comment:
Let's use the constructors. I.e.:

1. Create an FVM machine.
2. From the "system" actor, call the constructor on the reward, etc. actor code.